### ISSUE TITLE
Feature Request: Undo movement.

### ISSUE THREAD

**Issue by uLotUu** [manuelVo/foundryvtt-drag-ruler#57]
With this new cool feature to remember movement made in a current turn it would be super helpfull for a DM to have an "undo" option if a player moved by mistake. So it would reset both movement and the position of a token.

**Comment by manuelVo** [comment]
Until this is implemented into Drag Ruler you can use this Macro to reset the movement of the token that has the currently has it's turn. This macro can be executed by both the GM or the players (but will only work if a gm is connected):
```javascript
const combat = game.combats.active;
const combatant = combat.turns[combat.turn];
const flags = {trackedRound: 0, rulerState: null};
import("/modules/drag-ruler/src/socket.js").then(module => module.updateCombatantDragRulerFlags(combat, combatant, flags));
```

**Comment by manuelVo** [comment]
This is implemented in 70b166d8
